store/tikv/store_test.go

```diff
@@ -303,7 +303,8 @@ func (s *testStoreSuite) TestRequestPriority(c *C) {
 
 func (s *testStoreSerialSuite) TestOracleChangeByFailpoint(c *C) {
 	defer func() {
-		failpoint.Disable("github.com/pingcap/tidb/store/tikv/oracle/changeTSFromPD")
+		// to fix: err not nil
```

Comment:
because the failpoint is disabled already. I guess here we can first check the failpoint's status then disable it if need.

Reply:
So can the comment be removed now?